test/functional/http/get/transactions.js

```diff
@@ -1,13 +1,13 @@
 'use strict';
 
-var node = require('../node.js');
-var http = require('../common/httpCommunication.js');
-var ws = require('../common/wsCommunication');
-var sendLiskTrs = require('../common/complexTransactions.js').sendLISK;
-var transactionSortFields = require('../../sql/transactions').sortFields;
-var modulesLoader = require('../common/initModule').modulesLoader;
-var transactionTypes = require('../../helpers/transactionTypes.js');
-var genesisblock = require('../genesisBlock.json');
+var node = require('../../../node.js');
+var http = require('../../../common/httpCommunication.js');
+var ws = require('../../../common/wsCommunication');
+var sendLiskTrs = require('../../../common/complexTransactions.js').sendLISK;
+var transactionSortFields = require('../../../../sql/transactions').sortFields;
```

Comment:
Why is this stuff being initialized as part of each module? We should be init'ing modules and functions inside of the `node.js` file at the top level of the test structure.